db/models.py

```diff
@@ -41,8 +52,77 @@ def __str__(self) -> str:
 
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(to=CinemaHall, on_delete=models.CASCADE)
-    movie = models.ForeignKey(to=Movie, on_delete=models.CASCADE)
+    cinema_hall = models.ForeignKey(to=CinemaHall, on_delete=models.DO_NOTHING)
+    movie = models.ForeignKey(to=Movie, on_delete=models.DO_NOTHING)
```

Comment:
Do you really want to do nothing and leave movie session without hall or/and movie? It would make no sense to store this movie session and I believe it isn’t a good idea at all